client/README.md

```diff
@@ -2,8 +2,47 @@
 [![NPM Package](https://img.shields.io/npm/v/@stacks/blockchain-api-client.svg?style=flat-square)](https://www.npmjs.org/package/@stacks/blockchain-api-client)
 [![Build Status](https://github.com/blockstack/stacks-blockchain-api/workflows/Build/badge.svg)](https://github.com/blockstack/stacks-blockchain-api/actions)
```

Comment:
I left this broken image here initially. If you want to fix, it should be
```
https://github.com/blockstack/stacks-blockchain-api/workflows/stacks-blockchain-api/badge.svg
```

Reply:
👍